### PR TITLE
add response header about procesing time

### PR DESCRIPTION
**Pull request by AlvaroVega**
Twin PR https://github.com/telefonicaid/iotagent-json/pull/845

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- ADD: reponse header about procesing time (iotagent-node-lib#1650)
+
```
Comment (fgalan) [inline Changelog:2]:
```suggestion
```

CNR -> Changelog is done at releasing time

File: lib/constants.js [hunk 2]
```diff
@@ -53,5 +53,7 @@ module.exports = {
     AMQP_DEFAULT_QUEUE: 'iotaqueue',
     AMQP_DEFAULT_DURABLE: true,
     AMQP_DEFAULT_RETRIES: 5,
-    AMQP_DEFAULT_RETRY_TIME: 5
+    AMQP_DEFAULT_RETRY_TIME: 5,
+
+    X_PROCESSING_TIME: 'X-Processing-Time'
```
Comment (fgalan) [inline lib/constants.js:58]: It is based in some existing (pseudo)standard? Or it has been fully invented?

Comment (AlvaroVega) [reply lib/constants.js:58]: It just a new invented header. But we can use other...

Comment (fgalan) [reply lib/constants.js:58]: Maybe we should do a little searching to check is there is something similar we can reuse or inspire on...

Comment (AlvaroVega) [reply lib/constants.js:58]: Maybe X-Process-Time matches better

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -1 +1 @@
-
+- ADD: reponse header about processing time (iotagent-node-lib#1650)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Add: reponse header about processing time (iotagent-node-lib#1650)
```

File: CHANGES_NEXT_RELEASE [hunk 4]
```diff
@@ -1 +1 @@
-
+- Add: reponse header about processing time (iotagent-node-lib#1650)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Add: X-Processing-Time response header with processing time (in milliseconds) expended by current HTTP measure (iotagent-node-lib#1650)
```

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
Twin PR https://github.com/telefonicaid/iotagent-json/pull/845

**Pull request merged**